src/box/box.cc

```diff
@@ -2391,6 +2391,10 @@ box_process1(struct request *request, box_tuple_t **result)
 	struct space *space = space_cache_find(request->space_id);
 	if (space == NULL || box_check_process_rw(space) != 0)
 		return -1;
+	if (box_slice_is_over()) {
+		diag_set(ClientError, ER_SPACE_ITERATION_TIMEOUT);
```

Comment:
Please use `FiberSliceIsExceeded` error here. We want to have exactly the same error no matter if it's a long iteration over a space or in Lua code.

Reply:
done